web/modules/custom/farm_nfa/farm_nfa.install

```diff
@@ -273,3 +274,69 @@ function farm_nfa_update_9019(&$sandbox) {
   $process = Drush::drush(Drush::aliasManager()->getSelf(), 'config-import', ['--partial'], ['source' => $config_path]);
   $process->run();
 }
+
+/**
+ * save the taxonomy terms for forest_vegetation and forest_type taxonomy types.
+ */
+function farm_nfa_update_9020(&$sandbox) {
+  $vocab_terms = [
+    'forest_vegetation' => [
+      'Tropical High Forest',
+      'Woodland',
+      'Bush',
+    ],
+    'forest_type' => [
+      'natural',
+      'plantation',
+    ],
+  ];
+  foreach ($vocab_terms as $vocab => $terms) {
+    $current_vocab = $vocab;
+    // Term names to be added.
+    foreach ($terms as $term) {
+      $term = Term::create(array(
+        'parent' => array(),
+        'name' => $term,
+        'vid' => $current_vocab,
+      ));
+      $term->save();
+    }
+  }
+}
+
+/**
+ * save the taxonomy terms for forest purpose taxonomy type
```

Comment:
Coding standards - Comments should start with a capital letter and end with a full-stop.